playground/backend/internal/code_processing/code_processing_test.go

```diff
@@ -962,36 +1096,78 @@ func Test_runStep(t *testing.T) {
 		pipelineOptions      string
 		pipelineLifeCycleCtx context.Context
 		cancelChannel        chan bool
+		createExecFile       bool
 	}
 	tests := []struct {
-		name string
-		args args
-		code string
+		name           string
+		args           args
+		code           string
+		expectedStatus pb.Status
 	}{
 		{
-			name: "Test run step working without an error",
+			name: "Test run step working on python sdk",
 			args: args{
 				ctx:                  context.Background(),
 				cacheService:         cacheService,
-				pipelineId:           uuid.UUID{},
+				pipelineId:           uuid.New(),
 				isUnitTest:           false,
-				sdkEnv:               sdkEnv,
+				sdkEnv:               sdkPythonEnv,
+				pipelineOptions:      "",
+				pipelineLifeCycleCtx: context.Background(),
+				cancelChannel:        make(chan bool, 1),
+				createExecFile:       true,
+			},
+			code:           helloWordPython,
+			expectedStatus: pb.Status_STATUS_RUN_ERROR,
+		},
+		{
+			name: "Test run step working on go sdk",
+			args: args{
+				ctx:                  context.Background(),
+				cacheService:         cacheService,
+				pipelineId:           uuid.New(),
+				isUnitTest:           true,
+				sdkEnv:               sdkGoEnv,
+				pipelineOptions:      "",
+				pipelineLifeCycleCtx: context.Background(),
+				cancelChannel:        make(chan bool, 1),
+				createExecFile:       true,
+			},
+			code:           helloWordGo,
+			expectedStatus: pb.Status_STATUS_RUN_ERROR,
+		},
+		{
+			name: "Test run step without preparing files with code",
```

Comment:
success or fail?

Reply:
Fail, expected to get `Status_STATUS_RUN_ERROR` pipeline status